packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -228,21 +228,26 @@ abstract class OAuth2Client {
     const tokens = await TokenStorage.get();
     const accessToken = tokens && tokens.accessToken;
 
+    const body: StringDict<string | undefined> = {
+      client_id: clientId,
+    };
+    // This is needed to support Token Vault; the SDK may not have the token locally
     if (accessToken) {
```

Comment:
This change is what ensures Token Vault support. The `accessToken` is the only thing conditionally set, since the Proxy may be the one to set it as only it has access to it.